### PR TITLE
Prevent the use of the var keyword

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by n1474335** [comment]
Looks good. Happy to merge once this is building correctly. Try not to conflate too many changes into each pull request. I see you've started making some changes to `Gruntfile.js`. If these aren't related to simply removing the `var` keyword, put them in a separate PR.

**Comment by graingert** [comment]
@n1474335 the Gruntfile was broken for me, without this change when removing vars.

**Comment by graingert** [comment]
@n1474335 done

File: Gruntfile.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ module.exports = function (grunt) {
      * Compiles a production build of CyberChef into a single, portable web page.
      */
     function runInliner() {
-        var inlinerError = false;
```
Comment (graingert) [inline Gruntfile.js]: this var broke, without the callbacks working properly, so might as well fix them at the same time.

**Pull request merged**